src/components/Card.js

```diff
@@ -0,0 +1,128 @@
+// modules
+import { useState, useEffect } from "react";
+
+const Card = ({ addListItems, burgerProduct, handleModalOpen }) => {
+	// state
+	const [counter, setCounter] = useState(0);
+	const [subtotal, setSubtotal] = useState(0);
+
+	// destructure burger
+	const { name, imgUrl, price } = burgerProduct;
+
+	// When counter is changed - update subtotal
+	useEffect(() => {
+		setSubtotal(price * counter);
+	}, [counter]);
+
+	const handleAdd = (e) => {
+		setCounter(counter + 1);
+		handleUnclick(e);
+	};
+
+	const handleSubtract = (e) => {
+		if (counter > 0) {
+			setCounter(counter - 1);
+		}
+		handleUnclick(e);
+	};
+	const handleSubmit = (e) => {
+		e.preventDefault();
+
+		// check to see if user selected counter
+		if (counter > 0) {
```

Comment:
Gotta check for those user errors! Nicely done here with the addition of the alert to bring attention to the problem immediately. If they are verging on hangry, they need the quick solve.